.github/workflows/vast.yaml

```diff
@@ -379,7 +379,7 @@ jobs:
       - name: Upload Artifact to GCS
         if: ${{ github.event_name == 'push' || github.event_name == 'release' }}
         run: |
-          gsutil -m cp "$BUILD_DIR/$PACKAGE_NAME.tar.gz" "gs://${{ secrets.GCS_BUCKET }}"
+          gsutil -m cp "${{ env.BUILD_DIR }}/package/${{ env.PACKAGE_NAME }}.tar.gz" "gs://${{ secrets.GCS_BUCKET }}/$PACKAGE_NAME.tar.gz"
```

Comment:
I'm not sure whether we have any downstream jobs that depend on that name, so maybe we can even keep the filename from cpack?

Reply:
The proprietary plugin CI depends on it, but you can easily adjust that to use another name.

Reply:
Let's do that some other time.